gravitee-apim-console-webui/src/management/management.module.ajs.ts

```diff
@@ -702,9 +702,6 @@ graviteeManagementModule.config(($mdThemingProvider: angular.material.IThemingPr
 });
 graviteeManagementModule.config(($showdownProvider) => {
   $showdownProvider.setOption('tables', true);
-  $showdownProvider.loadExtension('highlightjs');
```

Comment:
believe me I couldn't find a simpler way 🙄 

the extensions seem to load autonomously even if they are not explicitly loaded here .
⚠️ If you know how to check each extension. Don't hesitate to tell me, I didn't really find it. I did several tests and I think it's good but I'm not sure

Reply:
Ok, let's go with that then, I 💯  trust you 👍🏻